libsrc/apple2/cgetc.s

```diff
@@ -51,10 +51,14 @@ put_caret:
 
         ; At this time, the high bit of the key pressed is set.
 :       bit     KBDSTRB         ; Clear keyboard strobe
-        .ifdef __APPLE2ENH__
+
+        .ifndef __APPLE2ENH__
+        bit     machinetype     ; Apple //e or more recent?
+        bpl     clear
+        .endif
```

Comment:
I remember that many years back when CH_F1 - CHF10 were introduced for the Apple II, there was a community member arguing that it isn't necessary to limit them to the apple2enh targetr because the Apple ][+ user can push his joystick button isntead. "Of course" this wasn't considered reasonable because the user maybe/likely...
* doesn't know
* doesn't have a joystick connetec in the first place
But _IF_ we now decide to move CH_F1 - CHF10 to the apple2.h _THEN_ there's no reason to keep the ][+ user who a) knows and b) has a joystock to successfully type them.